modin/pandas/test/dataframe/test_join_sort.py

```diff
@@ -527,6 +527,43 @@ def test_sort_values_with_string_index():
     df_equals(modin_df, pandas_df)
 
 
+@pytest.mark.skipif(
+    Engine.get() not in ["Ray", "Dask", "Unidist"],
+    reason="We only need to test this case where sort does not default to pandas.",
+)
```

Comment:
I wonder if this actually needs to be skipped, it should do no harm in testing sorting for all engines?..

Reply:
I think we should skip the tests because they aren't meaningful for non-partitioned engines. Also one of them uses `modin_df._query_compiler._modin_frame.column_widths`.